modules/virt-creating-single-pvc-template-storage-pool.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-single-pvc-template-storage-pool_{context}"]
+= Creating a single `PVCTemplate` storage pool in a host path provisioner custom resource
+
+Specifying a single `pvcTemplate` (a standard Kubernetes `PersistentVolumeClaimSpec`)  allows you to specify a persistent volume claim (PVC) template, which is mounted on the node at the where you want to create hostpath provisioner (HPP) volumes. Using single `pvcTemplate`s can improve I/O performance. By configuring a cluster where each node has an additional disk to store the HPP, you effectively distribute many large volumes over multiple smaller virtual volumes.
+
+.Procedure
+
+. Ensure that your nodes have an available blank disk or partition that can be discovered by the `LocalStorage` Operator (LSO).
```

Comment:
So technically the LSO is not needed, our primary use case does use the LSO, but anything that can create PVs will work. And I don't want to paint ourselves into saying you require LSO, since you don't. I would structure this as, here is how you use the pvcTemplate. And then below, as an addendum show how you can use the LSO to create PVs that the hpp can consume.

Reply:
see above.